src/env.ts

```diff
@@ -22,6 +26,7 @@ const env = envalid.cleanEnv(process.env, {
   NODE_PORT: envalid.port({ default: 9944 }),
   ENABLE_INDEXER: envalid.bool({ default: true }),
   USER_URI: envalid.str({ devDefault: '//Alice' }),
+  PROXY: nullableStr({ devDefault: '5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy' }), // needs to be nullable
```

Comment:
I think the semantics are confusing here. Can we rename to `PROXY_FOR`? Can you then also update the readme to make it explicit that this is the `ss58` address that transactions will be performed as